src/formats/SMI.cpp

```diff
@@ -717,7 +743,7 @@ void SMIFormat::write_next(const Frame& frame) {
         }
         auto not_hit = std::find(written.begin(), written.end(), false);
         auto current_atom = static_cast<size_t>(std::distance(written.begin(), not_hit));
-        write_atom(frame, written, current_atom, current_atom);
+        this->write_atom(frame, written, current_atom, current_atom);
```

Comment:
Is the use of `this` recommended for style now?

Reply:
I don't know, I was feeling like it =). What do you think?

Reply:
Doesn't really matter to me.